docs/scripting/trust_scripts/sys.loc.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: sys.loc
+---
+
+> sys.loc displays a system's [[loc]].
+
+### Security Level
+
+LOWSEC
+
+## Syntax
+
+### CLI
+
+```
+sys.loc
+```
+
+### Script
+
+```
+#ls.sys.loc()
+```
+
+### Parameters
+
+No known parameters.
+
+### Return
+
+Returns a string.
+
+#### CLI
+
+```
+>>sys.loc
+users.fakeloc123
```

Comment:
Not sure if we have one but we should have a consistent loc in style guide. Bit of a roaming thought.

Reply:
Agree, I brought this up too, I'll open an issue for that. 'user' is a reserved username but 'users' is a TRUST username and appears orange if someone were to like... copy paste it from the wiki to in-game or whatever. I'll describe the problem in detail in the issue